### PR TITLE
[v5.0] Stac 16551 aad tuning

### PR DESCRIPTION
**Pull request by fvlankvelt**
introduce a new section "Anomaly detection" under "Configure", to host details on how to export feedback, scale up/down and troubleshoot using the status UI.
Draft site: https://stackstate.gitbook.io/stackstate-docs-development/v/stac-16551-aad-tuning/

### CODE REVIEW

File: configure/anomaly-detection/scaling.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+The streams with an anomaly check have the highest priority in the AAD.  All anomaly health checks are updated on time when the number of **Checked streams** is higher than the number of **Streams with anomaly checks**.
+
+We can get these metrics from StackState using the CLI:
+
+```text
+sts metric get <multi_metrics_id> -t stackstate.spotlight_streams_with_anomaly_check -f anomaly_health_checks.csv
+sts metric get <multi_metrics_id> -t stackstate.spotlight_streams_checked -f checked.csv
+```
+
+Here `<multi_metrics_id>` is the ID for the "StackState Multi Metrics" datasource (`sts datasource list`).
```
Comment (jcshoekstra) [inline configure/anomaly-detection/scaling.md:47]: should we script this too? such that running the full thing in one command is possible?

File: configure/anomaly-detection/status_ui.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+description: StackState Self-hosted v5.0.x
+---
+
+# Autonomous Anomaly Detector Status UI
```
Comment (jcshoekstra) [inline configure/anomaly-detection/status_ui.md:5]: not clear to me if we want to document this, and if we do, which (review) gates need to be passed

Comment (jcshoekstra) [reply configure/anomaly-detection/status_ui.md:5]: or if we should clearly state that it is a development / alpha portal

Comment (jcshoekstra) [reply configure/anomaly-detection/status_ui.md:5]: ah I realize now this documentation already existed, and this is simply moving it

File: configure/anomaly-detection/scaling.md (hunk 1 above)
Comment (fvlankvelt) [reply configure/anomaly-detection/scaling.md:47]: AFAICS it is not possible to extract the ID programmatically (i.e. with a one-liner).  The cli produces a table of all datasources, from which the ID can be copied.

**Pull request merged**